src/css/custom.css

```diff
@@ -56,6 +56,10 @@ nav {
     padding-left: 1rem;
 }
 
+.navbar__items {
+    white-space: nowrap;
```

Comment:
Are you sure its white-space not empty-space or blank?

Source: [Article - Inclusive Language in IT Services](https://uwindsor.teamdynamix.com/TDClient/1975/Portal/KB/ArticleDet?ID=141931)

Reply:
https://cdn.discordapp.com/emojis/725426242702147595.webp?size=96&quality=lossless